typescript/infra/config/environments/testnet2/interchain/queries/addresses.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Were you planning on deploying on mainnet as well?

Reply:
yeah, post-review